src/pages/home/report/ReportActionCompose.js

```diff
@@ -168,24 +181,35 @@ class ReportActionCompose extends React.Component {
                             <>
                                 <AttachmentPicker>
                                     {({openPicker}) => (
-                                        <TouchableOpacity
-                                            onPress={(e) => {
-                                                e.preventDefault();
-
-                                                // Do not open attachment picker from keypress event
-                                                if (!e.key) {
-                                                    openPicker({
-                                                        onPicked: (file) => {
-                                                            displayFileInModal({file});
-                                                        },
-                                                    });
-                                                }
-                                            }}
-                                            style={[styles.chatItemAttachButton]}
-                                            underlayColor={themeColors.componentBG}
-                                        >
-                                            <Icon src={Paperclip} />
-                                        </TouchableOpacity>
+                                        <>
+                                            <TouchableOpacity
+                                                onPress={(e) => {
+                                                    e.preventDefault();
+                                                    this.setMenuVisibility(true);
+                                                }}
+                                                style={styles.chatItemAttachButton}
+                                                underlayColor={themeColors.componentBG}
+                                            >
+                                                <Icon src={Paperclip} />
+                                            </TouchableOpacity>
+                                            <CreateMenu
+                                                isVisible={this.state.isMenuVisible}
+                                                onClose={() => this.setMenuVisibility(false)}
+                                                onAttachmentPickerSelected={() => {
+                                                    setTimeout(() => {
```

Comment:
the attachment modal wasn't opening unless i added this. feels a bit weird so if there's a better way, please let me know.

Reply:
Yeah this is a bit of a hack, I'll look into it and try to get back to you by tommorow

Reply:
We decided to merge with this workaround. Another issue has been opened to improve the workaround.